src/libraries/System.Data.OleDb/src/OleDbDataReader.cs

```diff
@@ -1022,8 +1022,9 @@ public override Type GetFieldType(int index)
         {
             if (null != _metadata)
             {
-                // TODO-NULLABLE: Should throw if null (empty), though it probably doesn't happen
-                return _metadata[index].type.dataType!;
+                Type? fieldType = _metadata[index].type.dataType!;
```

Comment:
The `?` and `!` shouldn't both be necessary; you can remove one of them.

Reply:
yes, just noticed as well 😄 already updated